timeless/restaurants/table_shapes/views.py

```diff
@@ -73,10 +74,14 @@ def edit(id):
     )
 
 
-@bp.route("/delete/<int:id>", methods=["POST"])
-def delete(id):
-    """ Delete table shape with id """
-    table_shape = models.TableShape.query.get(id)
-    DB.session.delete(table_shape)
-    DB.session.commit()
-    return redirect(url_for("table_shape.list"))
+class Delete(views.DeleteView):
+    """
+    @todo #173:30min Refactor all deleting views to use `views.DeleteView`
```

Comment:
@vryazanov ``#312`` here, not ``#173``, since that is the originating ticket :)

Reply:
@amihaiemil you are absolutely right, thanks!